rllib/core/rl_trainer/torch/torch_rl_trainer.py

```diff
@@ -107,33 +104,31 @@ def build(self) -> None:
         # TODO (Kourosh): Instead of using _TorchAccelerator, we should use the public
         # api in ray.train but allow for session to be None without any errors raised.
         if self._use_gpu:
-            self._device = _TorchAccelerator().get_device()
+            if self._distributed:
+                self._device = _TorchAccelerator().get_device()
+            else:
+                self._device = torch.device(self._local_gpu_id)
```

Comment:
is this interpreted as an id or as an index of the GPU device?

Reply:
this is an index over available gpu devices

Reply:
I left a comment so that we can refer back to it